extension/options.js

```diff
@@ -31,6 +31,13 @@ const templateOption = {
    */
   default: true,
 
+  /**
+   * (Required)
+   * Hovertext that describes what the option does in more detail. This will be
+   * shown in the option panel when the user hovers over the question mark.
+   */
+  hovertext: 'a template for other options to follow',
```

Comment:
A few suggestions here:

  1. [CHANGE REQUESTED] Maybe this field should be called `description` or `tooltip` instead of `onhover`? The fact that the text is displayed on hover seems like a UI implementation detail. (And it's also something that could change, since the current implementation is neither accessible nor mobile-friendly.)

  2. [CHANGE REQUESTED] I recommend formatting the example text the same way that you want the tooltip to be formatted, i.e., with a capital letter at the start and a period in the end. That way, when someone copies the template and replaces it with their own stuff, they will be more likely to use a style consistent with the other options.
 
  3. Just a comment: it's a little unfortunate that the option label is defined in options.html while the hovertext/tooltip is defined in options.js. I understand why you did it (the tooltip is added dynamically in Javascript) but ideally they would both be in the same place for consistency. (No need to change this now, but it's something I might refactor later.)